0800-appendix-57793-vallee.Rmd

```diff
@@ -56,19 +56,10 @@ fpr::fpr_table_wshd_sum(site_id = my_site) %>%
 
 ```
 
-<br>
-
-Vallee Lake is
-`r fpr::fpr_my_bcfishpass(site = my_site, col_pull = total_lakereservoir_ha)`ha
-in size and modelling indicates an additional
-`r fpr::fpr_my_bcfishpass(col_pull = total_wetland_ha)`ha of wetland
-upstream of the highway. Large lakes within watersheds can throttle high
-freshet and storm flows, increasing stability downstream for egg
-incubation and juvenile rearing.
 
 <br>
 
-Further downstream, GIS modelling indicates there is one crossing (modelled crossing ID 1802482) before the confluence of the Bulkley River. However, local knowledge from a landowner revealed that the crossing was no longer present. Upstream of crossing 57793, review of GIS indicates there are three crossings located on the mainstem of Vallee Creek, along with several modelled crossings on tributaries that are likely very small. PSCIS crossing 57801 is located past Vallee Lake on Highway 16 and was listed as a barrier at the time of survey by @gollner2013 in 2012. 
+Further downstream, GIS modelling indicates there is one crossing on a transmission line (modelled crossing ID 1802482). A local landowner reported that this crossing does not exist. Review of GIS indicates there is a modelled crossing (ID 1800041) located on the mainstem approximately 5.5km upstream of `r as.character(my_site)`, on Vallee Creek Road. Additionally, there are several modelled crossings upstream of subject site 57793, located on tributaries that are likely very small. A total of `r fpr::fpr_my_bcfishpass(col_pull = total_lakereservoir_ha)`ha of lake and `r fpr::fpr_my_bcfishpass(col_pull = total_wetland_ha)`ha of wetland is modelled upstream. These lakes include Vallee Lake(10ha), Helen Lake(16ha) and other small unnamed lakes that are of insignificant size. Upstream of crossing 1800041, the mainstem drains into Vallee Lake before flowing through the culvert (PSCIS crossing 57801) located on Highway 16. This crossing was listed as a barrier at the time of survey by @gollner2013 in 2012. Comments indicate that fish were observed downstream, and good habitat was present upstream. Vallee Creek then flows east past the highway through PSCIS crossings 57786, followed by 57787, before draining into Helen Lake. 
```

Comment:
the mainstem flows over the highway then into Vallee Lake - then through  1800041 then to our site. 


the statement that "the mainstem drains into Vallee Lake before flowing through the culvert (PSCIS crossing 57801) located on Highway 16" is backwards. It flows downstream from the highway to the Bulkley.  I can fix